content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -14,201 +19,184 @@ the *bill control* policy chosen in the *Purchase* app settings.
-:guilabel:`Vendor` field, and add products to the :guilabel:`Product` lines by clicking
-:guilabel:`Add a line`.
+Doing so opens a new Request for Quotation (RfQ) form. On the blank |RfQ| form, add a
+:guilabel:`Vendor`, and click :guilabel:`Add a line` under the :guilabel:`Product` tab to add
+products to the order.
 
-Then, confirm the :abbr:`RFQ (request for quotation)` by clicking the :guilabel:`Confirm Order`
-button above the detail form. Doing so turns the :abbr:`RFQ (request for quotation)` into a purchase
-order.
+Once ready, click :guilabel:`Confirm Order` to confirm the |RfQ| into a |PO|.
 
-Then, click the :guilabel:`Create Bill` button to create a vendor bill for the purchase order.
+Then, click :guilabel:`Create Bill` to create a vendor bill. This opens a :guilabel:`Draft Bill`
+form. From here, add a billing date in the :guilabel:`Bill Date` field.
 
-Clicking the :guilabel:`Create Bill` button reveals the :guilabel:`Draft Bill` page for the purchase
-order.
-
-On the :guilabel:`Draft Bill`, click the :guilabel:`Edit` button to modify the bill, and add a bill
-date in the :guilabel:`Bill Date` field. If needed, add additional products to the
-:guilabel:`Product` lines by clicking :guilabel:`Add a line` in the :guilabel:`Invoice Lines` tab.
-
-Next, confirm the bill by clicking the :guilabel:`Confirm` button on the :guilabel:`Draft Bill`
-page.
+Once ready, confirm the bill by clicking :guilabel:`Confirm` on the :guilabel:`Draft Bill` page.
 
 .. tip::
    Since the bill control policy is set to *ordered quantities*, the draft bill can be confirmed as
    soon as it is created, before any products have been received.
 
-On the new :guilabel:`Vendor Bill`, add a :guilabel:`Bill Reference` number, which can be used to
-match the bill with additional documents (such as the :abbr:`PO (purchase order)`). Then, click
-:menuselection:`Confirm --> Register Payment`. Doing so causes a pop-up to appear, wherein a payment
-:guilabel:`Journal` can be chosen; a :guilabel:`Payment Method` selected; and a :guilabel:`Recipient
-Bank Account` can be selected from a drop-down menu.
+On the new :guilabel:`Vendor Bill`, add a :guilabel:`Bill Reference` number. This number can be used
+to match the bill with additional documents, such as the |PO|.
+
+Once ready, click :guilabel:`Register Payment`.
+
+Doing so causes a :guilabel:`Register Payment` pop-up window to appear, wherein a payment
+:guilabel:`Journal` can be chosen, and a :guilabel:`Payment Method` selected.
 
 Additionally, the bill :guilabel:`Amount`, :guilabel:`Payment Date`, and :guilabel:`Memo`
-(:dfn:`Reference Number`) can be changed from this pop-up. Once ready, click :guilabel:`Create
-Payment` to finish creating the :guilabel:`Vendor Bill`. Doing so causes a green :guilabel:`In
-Payment` banner to display on the :abbr:`RFQ (request for quotation)` form.
+(:dfn:`Reference Number`) can be changed from this pop-up window, if necessary.
```

Comment:
I understand that you're going through a flow, so right on this line might not be the best place, but in addition to the info you have here, it could be really helpful to define every field on a vendor bill, including those in the **Journal Items** and **Other Info** tabs. This way, readers can rely on your doc for all the details they need to complete their vendor bills.

For example, it would be great to explain the difference between the Bill Date and Accounting Date. Also, fields like Payment Reference, Recipient Bank, and Payment Terms aren't mentioned, and providing details on these would be very beneficial. Just my thoughts!